packages/xo-server/src/api/vm.mjs

```diff
@@ -662,6 +676,11 @@ export const set = defer(async function ($defer, params) {
     await xapi.call('VM.set_suspend_SR', VM._xapiRef, suspendSr === null ? Ref.EMPTY : suspendSr._xapiRef)
   }
 
+  const uefiMode = extract(params, 'uefiMode')
+  if (uefiMode !== undefined) {
+    await xapi.call('VM.set_uefi_mode', VM._xapiRef, uefiMode)
+  }
+
```

Comment:
Why this change?

Reply:
Nothing changed it's just the indentation

Reply:
No, this code is new in this PR.

Reply:
Still there.

Reply:
`VM.set_uefi_mode` is called when we click on `Copy the pool's default UEFI certificates to the VM`.
See the [spec](https://github.com/vatesfr/xen-orchestra/issues/7495) made by Sam